setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup
 
-with open('version.txt') as f:
+with open('VERSION') as f:
```

Comment:
```suggestion
with open('version.txt') as f:
```

Reply:
ohoh :D i'll change it

Reply:
Or change github actions if you want to keep the name of the file as `VERSION`

Do delete that file and add it to `.gitignore` though so that it doesn't end up in this repo